### PR TITLE
feat: cca integrate upload scripts

### PR DESCRIPTION
**Pull request by loweisz**
Adds `upload` and `upload-ci` to the cca template. 
Explains the commands in the readme of the template

**Note:** This branch is bound to the canary release `v0.6.1-alpha.9+69bcb5c`

### CODE REVIEW

File: packages/@contentful--cra-template-create-contentful-app/template.json [hunk 1]
```diff
@@ -17,9 +17,15 @@
...
       "cross-env": "^7.0.3"
     },
+    "devDependencies": {
+      "@contentful/app-scripts": "^0.5.0"
+    },
     "scripts": {
-      "start": "cross-env BROWSER=none react-scripts start"
+      "start": "cross-env BROWSER=none react-scripts start",
+      "upload": "contentful-app-scripts upload --bundle-dir ./build",
+      "upload": "contentful-app-scripts upload --bundle-dir ./build",
```
Comment (kdamball) [inline packages/@contentful--cra-template-create-contentful-app/template.json:26]: Duplicate?

File: packages/@contentful--cra-template-create-contentful-app/template/README.md [hunk 2]
```diff
@@ -20,6 +20,19 @@ It correctly bundles React in production mode and optimizes the build for the be
...
+#### `npm run upload`
+
+Uploads the build folder to contentful and creates a bundle that is automatically activated.
+The command guides you through the deployment process and asks for all required arguments.
+Read [here](https://www.contentful.com/developers/docs/extensibility/app-framework/create-contentful-app/#deployment) for more information about the deployment process.
+
+#### `npm run upload-ci`
+
+Similar to `npm run upload` it will upload your app to contentful and activate it. The only difference is   
+that with this command all required arguments are read from the environment variables, for example when you add
```
Comment (kdamball) [inline packages/@contentful--cra-template-create-contentful-app/template/README.md:32]: Maybe also list all the required env vars? Assuming someone will just copy paste this

Comment (loweisz) [reply packages/@contentful--cra-template-create-contentful-app/template/README.md:32]: Good idea ! added it :)

**Review by kdamball — COMMENTED** [review]
Is there a reason `test-proj2` folder is added in there?

**Comment by loweisz** [comment]
> Is there a reason `test-proj2` folder is added in there?

it's Friday :D I forgot to remove it

**Review by kdamball — APPROVED** [review]

**Pull request merged**